workflows/taxprofiler.nf

```diff
@@ -17,7 +17,26 @@ def checkPathParamList = [ params.input, params.databases, params.hostremoval_re
 for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true) } }
 
 // Check mandatory parameters
-if (params.input    ) { ch_input     = file(params.input)     } else { exit 1, 'Input samplesheet not specified!' }
+if (params.input) {
+    ch_input = file(params.input)
+    ch_input_basedir = []
+
+} else if (params.pep) {
+
+    if ( params.pep.startsWith("http://") || params.pep.startsWith("https://") ) {
```

Comment:
As discussed at the module level, this shouldn't be necessary - NXF should detected it for you ;)